user_manual/groupware/sync_android.rst

```diff
@@ -18,6 +18,11 @@ Files and notifications
    **Security**. Generate an App password, click "Generate QR code" and
    tap the QR scanner icon in the Nextcloud app, point your phone's
    camera towards the screen.
+3. Once you are signed in you can view and edit files from within the app similar to how you would view and edit the files from within a browser. 
+4. You can sync files for offline access by clicking on the three dots to the right of the file name and choosing "download" or "sync".   
+   As it stands Syncing is only partially automated, so if you want your files to remain in sync you will need to click the sync button when
+   changes are made locally or on the nextcloud server. Clicking "sync" on a foulder will download and sync all files in that foulder, but 
```

Comment:
```suggestion
   changes are made locally or on the Nextcloud server. Clicking "sync" on a folder will download and sync all files in that folder, but 
```

Reply:
Partly right, partly wrong.
All synced files(!) stay in sync, as they are updated when on wifi.
This is not possible for folders, so folders needs to be synced manually (and then it also downloads new files)